docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,381 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Best practices from Cisco recommand to extend the URL length to avoid issue with longer request. 
+
+To do so, enter the following command in the CLI on the NDES server:
+
+ %systemroot%\system32\inetsrv\appcmd.exe set config /section:system.webServer/
+ security/requestFiltering /requestLimits.maxQueryString:"16384" /commit:apphost
+
+Certificate template
+^^^^^^^^^^^^^^^^^^^^
+
+NOTE: Remember that the validity of your CA can impact your whole certificate architecture.
+
+The main goal is to deliver certificate for `user Authentication`, this mean we have to setup a specific template.
+
+First, the certificate template need at least the following `Enhanced Key Usage` and `Key Usage`:
+
+image::docs/images/scep-ms-pki-en-key-usage.png[scaledwidth="100%",alt="Mandatory Enhanced Key Usage"]
+
+image::docs/images/scep-ms-pki-key-usage.png[scaledwidth="100%",alt="Mandatory Key Usage"]
+
+Since we will duplicate a template, those `Key Usage`, and `Enhanced Key Usage` are already configured. We advise to duplicate the template `User` and change the necessary settings.
+
+To duplicate the template, you need to navigate through `Server Manager` > `Roles` > `Active Directory Certificates Services` > `Certificate templates`. Now right click the template `User` and select `Duplicate this template`.
+
+Once duplicated, right click your new template, go to `Properties`. Navigate to the tab `Subject Name`. Make sure to select `Supplied in the request` over `Built from information in Active Directory`, else the requested CN will be overwritten by NDES.
+
+To allow NDES to use this template you need to navigate to `Server Manager` > `Roles` > `Active Directory Certificates Services` > expand `YourServerName`, right click `Certificate template` and chose `New template to issue`, in the list select your newly created template.
+
+Now that you chose the template to deliver you need to configure it in the registry.
+
+For the registry, press `Start` and type `regedit` in the search bar.
+
+While in the registry navigate to `Computer` > `HKEY_LOCAL_MACHINE` > `SOFTWARE` > `Microsoft` > `Cryptography` > `MSCEP`.
+
+Here you should have a list of three keys entries: 
+
+* EncryptionTemplate,
+* GeneralPurposeTemplate,
+* SignatureTemplate.
+
+Replace each value with our newly created template. The default value should be `IPSECIntermediateOffline`.
+
+You now need to reboot the NDES server to apply changes to the registry.
+
+IIS configuration
+^^^^^^^^^^^^^^^^^
+
+The use of SCEP with PacketFence also require a change in the IIS configuration.
+
+Navigate to `Server Manager` > `Web(IIS)` > expand `Default web site` > select `CertSrv` > `mscep`. Select `Authentication`, and double click `Anonymous Authentication`. Make sure that `Application pool identity` is selected.
+
+Online Certificate Status Protocol (OCSP)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For the configuration of OCSP, there are small changes to apply.
```

Comment:
the use of small is useless. Instead you could say

For the configuration of OCSP, the following changes are necessary